pcdsdevices/epics/slits.py

```diff
@@ -106,10 +207,28 @@ def unstage(self):
         return super().unstage()
 
     def open(self):
+        """
+        Open the slits to have a 50mm aperature on each side
```

Comment:
docstring typo?

Reply:
MFX is all set to 50 (which is a terrible value). It is just the `OPEN.DOL*` fields that control the value so this is probably set to a more sane value in hutches where people use this record. I'll edit the docstring to not specify a value